Jenkinsfile.aws-test

```diff
@@ -22,13 +22,23 @@ node(NODE) {
         return
     }
 
+    // We're only ever triggered by the cloud job, so we know the latest build is in latest/
+    // We immediately resolve it back to the specific images/ dir
+    def dirpath, version
     try {
     utils.inside_assembler_container("-v /srv:/srv") {
     stage("Sync In") {
         withCredentials([
             string(credentialsId: params.ARTIFACT_SERVER, variable: 'ARTIFACT_SERVER'),
             sshUserPrivateKey(credentialsId: params.ARTIFACT_SSH_CREDS_ID, keyFileVariable: 'KEY_FILE'),
         ]) {
+            dirpath = "${images}/cloud/latest"
+            sh "mkdir -p ${dirpath}"
+            utils.rsync_file_in(ARTIFACT_SERVER, KEY_FILE, "${dirpath}/meta.json")
```

Comment:
I think we could adapt it when we adapt the rest of this script, no? E.g. we could just `rsync` to the workspace instead of `/srv`. Or I might be misunderstanding what you mean.

Reply:
This job wouldn't be too hard to rework without `/srv` given that we're only syncing tiny JSON files. Let's do that in a follow-up patch?

Reply:
As is today it adds a risk that we'll clobber a `meta.json` built by the cloud job right?

Reply:
We're only rsync'ing out to one location though, right? (`${images}/aws-${AWS_REGION}-tested.json`). And that file is owned by this job.

Reply:
OK, whipped something up in :arrow_down:. The delta is now larger though... so I'll have to test this after all.